docs/usage/getting-started/running.md

```diff
@@ -1,70 +1,82 @@
+If you're using the Mend Renovate App on GitHub, or if someone else is hosting Renovate for you, skip ahead to the [installing & onboarding](./installing-onboarding.md) page.
 
 ## Self-Hosting Renovate
 
-Self-hosting Renovate means that you are the "administrator" of the bot, which entails:
+When self-hosting Renovate you're the "administrator" of the bot, this means you:
 
-- You provide infrastructure for it to run on,
-- You provision its global config,
-- You ensure it's running regularly,
-- You ensure Renovate bot itself is updated
+- provide the infrastructure that Renovate runs on,
+- provision Renovate's global config,
+- make sure Renovate bot runs regularly,
+- make sure Renovate bot itself is updated
 
-Read our section on [Self-hosting on Windows](./installing-onboarding.md#self-hosting-on-windows) to prevent line endings from confusing Renovate bot.
+If you're self-hosting Renovate on Windows, read [Self-hosting on Windows](./installing-onboarding.md#self-hosting-on-windows) to prevent line endings from confusing Renovate bot.
 
 ### Available distributions
 
 #### npm package (CLI)
 
 Renovate's Open Source CLI is built and distributed as the npm package `renovate`.
-You can run this directly in any Node.js environment - even via `npx` - and it will process all the repositories it is configured with, before exiting.
-When you install Renovate from npm it naturally does not come bundled with any third-party tools or languages such as Ruby, Python, Composer, Bundler, Poetry, etc.
+You can run this package in any Node.js environment - even via `npx` - and it will process all the repositories it is configured with, before exiting.
+
+Renovate's npm package does not bundle any third-party tools or languages like Ruby, Python, Composer, Bundler, Poetry, etc.
 If you need Renovate to support any non-npm lock files like Bundler then you'll need to make sure all required third-party tools are pre-installed in the same environment alongside Renovate before you run it.
 
 The `renovate` npm package is compatible with all of Renovate's supported platforms.
 
-Renovate requires Node.js `>=14.15.0` and Git `>=2.33.0`.
+Renovate requires Node.js `>=18.12.0` and Git `>=2.33.0`.
+
+#### Docker images
+
+The `renovate` npm package is also distributed via Node.js images on Docker Hub (`renovate/renovate`).
+These Docker images work on all the platforms that Renovate supports.
+
+There are two flavors:
 
-#### Docker image
+- The default "full" image, this is the image you'll get if you use the `latest` tag
+- The `-slim` image
 
-The `renovate` npm package is also distributed via pre-built Node.js images on Docker Hub (`renovate/renovate`).
+##### The default image
 
-The `slim` image contains only Node.js so works if either:
+The default image comes with every package manager that Renovate supports.
+This makes the default image the best choice for most users.
```

Comment:
i think this is now wrong. as both images are using install mode by default, so they always install each manager on the fly. they are only skipped when the same version is already installed.

so I would prefer to suggest the slim image.

Reply:
This means the pros/cons of the `slim` and default images change. It sounds like both images install the needed managers on the fly.

What are the tradeoffs that you make when you choose `slim` vs default image? I imagine getting the `slim` image, and then downloading 20 package managers is worse than getting the full fat default image, for example?

Can you comment what kind of benefits/drawbacks each choice has? I'll then edit the docs to put those comparisons in the docs. 😄

Reply:
The only significant reason for the full image is for people who don't want to download/install things at runtime?

Reply:
sure, bit they also need to change to global mode. otherwise renovate will still install latest version of the tools or the requested version from repo. and the preinstalled tool are sometimes outdated if the user isn't updating the docker image regularly.


we also support a persistent cache for downloaded tools, so that we only need to unpack on second+ run.

Reply:
Maybe the full image should default to binarySource=global as a breaking change in a next major release?

Reply:
🤷‍♂️

Reply:
> so I would prefer to suggest the slim image.

> The only significant reason for the full image is for people who don't want to download/install things at runtime?

Should we make `-slim` the default version you get when you use the `latest` tag then?

Reply:
I don't think we should be holding up this PR while discussing such a big change like that

Reply:
Here's the issue to discuss my idea: https://github.com/renovatebot/docker-renovate/issues/1237

Reply:
> Maybe the full image should default to binarySource=global as a breaking change in a next major release?

Here's the issue for that idea: 😄 

- https://github.com/renovatebot/docker-renovate-full/issues/1599

Reply:
Ok, but i think we should still suggest to use the slim image by default